src/crump/type_detection.py

```diff
@@ -25,6 +25,10 @@ def detect_column_type(values: list[str]) -> str:
     if not non_empty:
         return "text"
 
+    # Check if all values are bigints (large integers)
+    if all(_is_bigint(v) for v in non_empty):
```

Comment:
This will fail if some rows are integer and some are big integer - if they are all at least integer but any are bigint then the column needs to be bigint